tensorhive/app/web/AppServer.py

```diff
@@ -78,7 +80,7 @@ def start_server():
             'workers': APP_SERVER.WORKERS,
             'loglevel': APP_SERVER.LOG_LEVEL
         }
-        log.info(green('[✔] Web App available at: http://{}:{}'.format(API.URL_HOSTNAME, APP_SERVER.PORT)))
+        log.info(green('[✔] Web App available at: http://{}:{}'.format(API_SERVER.HOST, APP_SERVER.PORT)))
```

Comment:
```suggestion
        log.info(green('[✔] Web App available at: http://{}:{}'.format(APP_SERVER.HOST, APP_SERVER.PORT)))
```

Reply:
I think that the webapp hostname should be displayed instead of api hostname here. Can somebody confirm?